tests/integration/components/flexberry-field-test.js

```diff
@@ -54,18 +75,18 @@ test('label mode works properly', function(assert) {
 
   // Check that label attribute doesn't exist now.
   this.set('label', null);
-  assert.strictEqual(this.get('label'), null, 'Component\'s hasn\'t wrapper is a <lable>');
+  assert.strictEqual(this.get('label'), null, 'Component\'s hasn\'t wrapper is a <label>');
```

Comment:
Typo fixed, but assert comment is still strange. More siutable is "Component hasn't inner `<label>`".